<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirNavBar</class>
 <widget class="QWidget" name="DirNavBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="toolButton_6">
     <property name="toolTip">
      <string>Bookmarks</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../nullfxp.qrc">
       <normaloff>:/icons/folder-favorites.png</normaloff>:/icons/folder-favorites.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Clear...</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_5">
     <property name="toolTip">
      <string>Go Home</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../nullfxp.qrc">
       <normaloff>:/icons/user-home.png</normaloff>:/icons/user-home.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_4">
     <property name="toolTip">
      <string>Go Previous</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../nullfxp.qrc">
       <normaloff>:/icons/go-previous.png</normaloff>:/icons/go-previous.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_3">
     <property name="toolTip">
      <string>Go Next</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../nullfxp.qrc">
       <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_2">
     <property name="toolTip">
      <string>Go Parent</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../nullfxp.qrc">
       <normaloff>:/icons/go-up.png</normaloff>:/icons/go-up.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton">
     <property name="toolTip">
      <string>Reload</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../nullfxp.qrc">
       <normaloff>:/icons/appointment-recurring.png</normaloff>:/icons/appointment-recurring.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_7">
     <property name="toolTip">
      <string>Zoon In/Out View</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../nullfxp.qrc">
       <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../nullfxp.qrc"/>
 </resources>
 <connections/>
</ui>
